composer.json

```diff
@@ -132,6 +134,10 @@
             "Composer\\Config::disableProcessTimeout",
             "paraunit run --testsuite integration"
         ],
+        "test:mutation": [
+            "Composer\\Config::disableProcessTimeout",
+            "infection --threads=max --only-covered --min-covered-msi=80"
```

Comment:
Shouldn't the CI command also use `--min-covered-msi=80`?

Reply:
No, here we have average MSI calculated for all tests, but in PRs we run mutation tests only for modified classes and it may happen that they have lower MSI. In such cases, we only want to annotate code with escaped mutants, but we don't want to fail the job and force PR's author to improve MSI 🙂. For now it's strictly informative tool, that should help us with verifying quality of the tests in each PR.

On the other hand, `test:mutation` is for determining MSI in project, we can think about running it as a scheduled workflow at some point to observe the metric 🙂. But it's unlikely to be added in regular PRs.